src/lib/collab/schema.js

```diff
@@ -29,11 +29,57 @@ import { Schema } from 'prosemirror-model'
 //https://github.com/cozy/cozy-notes/pull/2/commits/202e529cdb1e71996c2cab43057984c9f885b61a
 
 export const nodes = [
+  [
+    'date',
+    {
+      inline: true,
+      group: 'inline',
+      selectable: true,
+      attrs: {
+        timestamp: {
+          default: ''
+        }
+      },
+      parseDOM: [
+        {
+          tag: 'span[data-node-type="date"]'
+        }
+      ]
+    }
+  ],
+  [
+    'status',
+    {
+      inline: true,
+      group: 'inline',
+      selectable: true,
+      attrs: {
+        text: {
+          default: ''
+        },
+        color: {
+          default: ''
+        },
+        localId: {
+          default: '046b16b8-790a-4097-881d-eb377338287b'
```

Comment:
Ca sert à quoi ça pour ma culture ?

Reply:
Bonne question, je viens de regarder et celui-là je le retrouve nul part... 

Mais en gros puisque j'ai eu des erreurs sur le schema voici vite fait ce que j'ai compris : 
- prosemirror s'attend à pouvoir générer lui-même des nodes en respectant les contraintes 
- si les contraintes sont trop dynamiques et que prosemirror ne peut pas générer par lui-même des éléments, il va refuser le schema
- cela pour préserver l'intégrité du contenu et être sur d'avoir des éléments corrects par la suite et non pas juste  des bouts 

(cf https://discuss.prosemirror.net/t/how-to-create-a-nodetype-consist-of-an-image-and-text-with-inline-editing-experience/1326/2 . J'ai dû désactiver les "layout" car j'avais fait erreur. Je me suis noté de la regarder plus tard pour voir si on peut y faire qq chose) 

J'imaginais que ce default était là pour ça, histoire d'avoir la même forme qu'un vrai élément, mais je vois aucun localId dans les éléments que je créé manuellement 🤔 

Je t'avouerai que je ne suis pas allé voir l'implémentation de cet object, je sais que j'aurais besoin d'aller regarder tout ça quand on ajoutera les images dans les notes, je me réserve cette grosse partie à ce moment là.